al-sufi-arabic/description.md

```diff
@@ -0,0 +1,160 @@
+##### Ophiuchus
+The man holding the snake.
+##### Serpens
+The serpent.
+##### Sagitta
+The arrow.
+##### Aquila
+The eagle.
+##### Delphinus
+The Dolphin.
+##### Equuleus
+The Horse part.
+##### Pegasus
+The great horse.
+##### Andromeda
+The chained woman.
+##### Triangulum
+The triangle.
+##### Aries
+The ram.
+##### Taurus
+The bull.
+##### Gemini
+The twins.
+##### Cancer
+The crab.
+##### Leo
+The lion.
+##### Virgo
+The virgin.
+##### Libra
+The balance.
+##### Scorpius
+The scorpion.
+##### Sagittarius
+The archer.
+##### Capricornus
+The kid.
+##### Aquarius
+The water spiller.
+##### Pisces
+The two fishes.
+##### Cetus
+The monster Cetus.
+##### Orion
+Al-Jabbar (Arabic for the king).
+##### Eridanus
+The river.
+##### Lepus
+The rabbit.
+##### Canis Major
+The large dog.
+##### Canis Minor
+The little dog.
+##### Argo Navis
+The ship.
+##### Hydra
+The snake.
+##### Crater
+The bowl.
+##### Corvus
+The crow.
+##### Centaurus
+Centaurus, a half man half beast creature.
+##### Lupus
+The beast.
+##### Ara
+The censer.
+##### Corona Australis
+The southern crown.
+##### Piscis Austrinus
+The southern fish.
+##### The Horse
+A constellation imagined and illustrated by Al-Şūfī near Pegasus.
+##### The great fish
+A constellation imagined and illustrated by Al-Şūfī in the area of Andromeda.
+##### The other fish
+Another fish constellation imagined and illustrated by Al-Şūfī in the area of  Andromeda.
+
+### Compilation of the sky culture
+
+This sky culture is a result of research done on manuscripts of Al-Şūfī book, oldest of which is the [Oxford, Bodleian Library Ms. Marsh 144 dated 1009 AD](https://iiif.bodleian.ox.ac.uk/iiif/viewer/c1caa84c-f6d2-483f-9eb4-2439cccdc801#?c=0&m=0&s=0&cv=25&r=0&xywh=-4815%2C-378%2C14782%2C7535).
+Ten manuscripts of the **The Book of the Stars** were consulted to produce the most accurate text as was written by Al-Şūfī. Then, each member star of the 48 constellations was identified according to Al-Şūfī description, and new tables were built including modern star designations from the Hipparcos, Bright stars, and Bayer catalogues. In addition, Arab indigenous star names were collected and identified as mapped by Al-Şūfī to the Ptolemaic ones. All the previous steps were included in a book published digitally of the **The Book of the Stars** plus explanations and commentary by Khalid Al-Ajaji, the author of this sky culture[^2].
+
+The sky culture stick figures of the constellations are the images in the Ulugh Beg manuscript of the **The Book of the Stars**, extracted, and adapted for Stellarium by Mr. Kutaibaa Akraa [^3].
+
+The single source of the sky culture is the **The Book of the Stars** and the aim is to give a faithful representation of the book content.
+
+## References
+1. IBN QUTAYBA, AD-DINAWARI, ABŪ MUḤAMMAD ‘ABDULLAH B. (died ~ 889 AD), KITĀBU'L-ANWĀ’ (on meteorolgy of the Arabs), (Arabic print of the original book, DAIRUT'L-MA‘ARIFI'L-OSMANIA, Hydarabad, India, 1956 AD)
```

Comment:
Please use this format for the references, they are actually parsed by the code (on mobile) but need to follow this exact format:

` - [#1]: Star Tales on [Ian Ridpath website](http://www.ianridpath.com/startales/startales1b.htm)`

Reply:
Shall I include the Arabic title as in the following, or just give atranslated title in English?

```
- [#1]: IBN QUTAYBA, AD-DINAWARI, ABŪ MUḤAMMAD ‘ABDULLAH B. (died ~ 889 AD), KITĀBU'L-ANWĀ’ (on meteorolgy of the Arabs), (Arabic print of the original book, DAIRUT'L-MA‘ARIFI'L-OSMANIA, Hydarabad, India, 1956 AD)

أبو محمد عبد الله بن مسلم ابن قتيبة الدينوري (توفي 276 هـ)، كتاب الأنواء في مواسم العرب، داءرة المعارف العثمانية، حيدر أباد، الهند، 1375 هـ

```

Reply:
Done. But I left the original Arabic title in a separate line.